mitodl/mitxpro
# Migrate from travis to github actions

#### Pre-Flight checklist


#### What are the relevant tickets?
Fixes #2016 

#### What's this PR do?
It removes all `Travis` and related settings from the app and migrates to `Github Actions`.

#### How should this be manually tested?
- Make sure relevant test suits are still running
- Make sure that all tests are working fine
- Coverage report still working fine

#### Where should the reviewer start?
(Optional)
